src/node.cc

```diff
@@ -2617,6 +2617,9 @@ static void PrintHelp() {
          "  --inspect[=[host:]port]    activate inspector on host:port\n"
          "                             (default: 127.0.0.1:9229)\n"
 #endif  // HAVE_INSPECTOR
+         "  --loader=file              (with --experimental-modules) use the \n"
```

Comment:
If I understand @devsnek's [comment](https://github.com/nodejs/node/pull/22104#issuecomment-412293396) correctly, this line causes the [parallel/test-cli-node-print-help.js](https://github.com/nodejs/node/blob/master/test/parallel/test-cli-node-print-help.js) to fail. Maybe we can replace `(with --experimental-modules)` with something else, like`(with experimental-modules flag)`?

Reply:
However, if this flag can be used only with `--experimental-modules`, it seems we should place it inside the same if-clause and update the ` parallel/test-cli-node-print-help.js`.

Reply:
i'm opening a pr to fix the flags. this line should stay how it is.